src/Driver/driver_configs.jl

```diff
@@ -355,7 +355,7 @@ function AtmosGCMConfiguration(
     ),
     mpicomm = MPI.COMM_WORLD,
     grid_stretching = (nothing,),
-    meshwarp::Function = cubedshellwarp,
+    meshwarp::Function = equiangular_cubed_shell_warp,
```

Comment:
@charleskawczynski in the case of an assignment like this one, how would it look with the new interface you showed me? 

Like 
```
meshwarp::Function = cubed_sphere_warp(EquiangularCubedSphere()),
```
with no additional arguments?

Reply:
Or maybe I figured! 
Here I would just keep the assignment as is (just renaming the function)

```
meshwarp::Function = cubed_sphere_warp,
```
And then later, when `meshwarp` is used, I would used it with the specialization?
But later on in the file I see it is only used as `meshwarp = meshwarp`

Reply:
I think this question might turn out to be a bit more complex than I initially thought. Since I have seen that this `meshwarp` is used in every occurrence of `DiscontinuousSpectralElementGrid` , and this is used a gazillion times. 
Can I have a pointer on how to solve this in a Julian way?
cc: @charleskawczynski @jkozdon